roles/docker_private_registry/tasks/main.yml

```diff
@@ -14,13 +14,26 @@
       path: /etc/containers/registries.conf
     register: reg
 
-  - name: add registeries
+  - name: Check for TOML formmated registries.conf
+    command: grep registries.search /etc/containers/registries.conf
+    register: toml
+    when: reg.stat.exists
+    ignore_errors: true
+
```

Comment:
I originally forgot to ignore errors on the grep so it failed when it wasn't a TOML file in the case of fedora/26/atomic.  Fixed it here.